magpy/stream.py

```diff
@@ -27,15 +28,17 @@
 
 # Logging
 # ---------
-# Select the home directory of the user (platform independent)
-from os.path import expanduser
-home = expanduser("~")
+# Select the user's home directory (platform independent) or environment path
+try:
+    path_to_log = os.environ["MAGPY_LOG_PATH"]
```

Comment:
- Make sure this value is not empty.
- If it's not empty, check whether this directory exists and create it and/or it's parent directories if needed